src/views/v2/widgets/events-list.php

```diff
@@ -59,6 +61,7 @@
 	'phone'     => true,
 	'organizer' => true,
 ];
+$view_more_link = '#';
```

Comment:
:warning: Equals sign not aligned with surrounding assignments; expected 5 spaces but found 1 space

<sub>hash: 73e200639cd2e31614ea3c32858022d1</sub>